<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawer</class>
 <widget class="QWidget" name="drawer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dchat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imagesrsc.qrc">
    <normaloff>:/素材/png-1205.png</normaloff>:/素材/png-1205.png</iconset>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>211</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="groupMember">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>319</height>
     </rect>
    </property>
    <attribute name="label">
     <string>group member</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>125</width>
       <height>317</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>123</width>
           <height>281</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="toolButton_1">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>zhaoxing</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/1.jpg</normaloff>:/images/1.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_7">
            <property name="minimumSize">
             <size>
              <width>68</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>yahei</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/6.jpg</normaloff>:/images/6.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_2">
            <property name="minimumSize">
             <size>
              <width>74</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Li dog</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/2.jpg</normaloff>:/images/2.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_3">
            <property name="minimumSize">
             <size>
              <width>121</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Ye liang chen</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/3.jpg</normaloff>:/images/3.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_5">
            <property name="minimumSize">
             <size>
              <width>49</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>JY</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/5.jpg</normaloff>:/images/5.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_6">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>yy</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/6.jpg</normaloff>:/images/6.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_9">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>fixed</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/8.jpg</normaloff>:/images/8.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_8">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>hei hei</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/9.jpg</normaloff>:/images/9.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_4">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>ui</string>
            </property>
            <property name="icon">
             <iconset resource="imagesrsc.qrc">
              <normaloff>:/images/7.jpg</normaloff>:/images/7.jpg</iconset>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="blackList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>319</height>
     </rect>
    </property>
    <attribute name="label">
     <string>black list</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imagesrsc.qrc"/>
 </resources>
 <connections/>
</ui>
